### DISCUSSION TITLE
5기 3주차 회고

### DISCUSSION THREAD

**Discussion by imakerjun** [next-step/blog#254] [회고]
```
# 📅 4월 27일~5월 3일의 회고
## 📝 블로그 글 소개

## 💡 인사이트 있었던 일

## 🤔 아쉬움이 남은 일

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
```

**Comment by oen142** [comment]
# 📅 4월 27일~5월 3일의 회고
## 📝 블로그 글 소개

- 최근 서버 배포건으로 cafe24 땡 리눅스에 서버 셋팅을 진행해보았습니다.
- 갑자기 안되는것도 있고 버전 마이그레이션하면서 알아낸것들을 적어보았습니다.


## 💡 인사이트 있었던 일

- 셀리눅스  덕분에 nginx 포트포워딩이 실패해서 한참 찾다가 해결 방법은 찾았는데 맞는지 몰라서
-  좀더 적용해보고 블로그 글로 남겨야지 싶음
- 최근 패캠에서 하도 광고해서 듣는 강의가 있는데 나름 저렴한(?) 가격으로 여러개 배워서 듣는중

## 🤔 아쉬움이 남은 일
- 패캠이 좀더 일찍 열었으면 빨리 배워서 실무에 적용해볼수 있었을텐데 아쉽당.

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
- 패캠 강의 완강

**Comment by seovalue** [comment]
# 📅 4월 27일~5월 3일의 회고
## 📝 블로그 글 소개
* 우테코 모의면접에서 자바는 call-by-value인지, call-by-reference인지에 대한 질문을 받았습니다.
* 몰라서 레퍼런스로 찍었는데, 찍은 것도 들키고 심지어 틀렸습니다.
* 충격을 받아 질문해주신 검프에게 여쭈어가고, 레퍼런스들을 참고하며 (다음엔 제대로 답변하기 위해) 글을 작성해보았습니다.

## 💡 인사이트 있었던 일
* 회고는 처음 참여해보는데, 다른 분들의 회고록을 읽으니 주간 회고를 다시 시작해봐야겠다는 생각이 들었습니다.

## 🤔 아쉬움이 남은 일
* 요즘 무언가 머릿속이 정리되지 않는 느낌이 강해서, 학습을 해도 튕겨나가는 느낌이 들곤 했습니다.
* 이번 주엔 극복해보도록..!

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
* 1일 1기술포스팅은 유지하되, 일요일엔 주간 회고를 남기기!

**Comment by yujo11** [comment]
# 📅 4월 27일~5월 3일의 회고
## 📝 블로그 글 소개
- '함께 자라기'라는 책을 읽고 서평을 작성하는 중입니다. (현재 미완)
- 책을 읽으면서 많은 인사이트를 받았지만 그걸 글로 풀어내는건 어렵네요😅😅

## 💡 인사이트 있었던 일
1. 페이스북에서 얻은 인사이트
```
* 쉬운 일을 어렵게 만드는 방법: 미룬다.
* 어려운 일을 쉽게 만드는 방법: 함께한다.
* 없던 스트레스를 만드는 방법: 걱정한다.
* 있던 스트레스도 없애는 방법: 시작한다.
```
- 평소라면 이런 글을 제대로 읽지도 않고 넘겼겠지만 얼마 전 번아웃을 겪으면서 왔던 현타를 정확히 짚어주는 글이라 몇번 다시 읽게 됐습니다.

2. 페어에게서 얻은 인사이트
- 페어프로그래밍을 함께 진행한 페어가 UI/UX에 굉장히 관심이 많았습니다.
- 같이 페어프로그래밍을 하면서 UI/UX에 대한 관점, 웹 접근성에 대한 관점에서 많은걸 배울 수 있었습니다.

## 🤔 아쉬움이 남은 일
- 지난 일들에 대한 후회로 현재에 집중하지 못한 일

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
- 매일매일 하루에 집중하며 살기

**Comment by catsbi** [comment]
# 📅 4월 27일~5월 3일의 회고
## 📝 블로그 글 소개
- 2021년 4월 5주차 한주간 있었던 일을 정리하는 회고록입니다. 
    - 우아한 테크코스 프로 프리코스를 진행하며 생각하던 고민들과 이를 해결하기위해 했던 내용들에 대해 작성해보았습니다.
    - IntelliJ의 새 기능 Code With Me를 사용하며 느낀점
    - 자바의 정석 컬렉션 프레임워크 발표준비를 하며 작성한 포스팅과 그 포스팅을 작성하며 느낀점
    - 식단관리
    - 새로운 강의(JUnit5)
    - 5월에 진행할 과정
    
## 💡 인사이트 있었던 일
: 대학교 동기이자 친한친구가, 지금 대학교에서 교수로 있는데, 개발을 진행하며 느낀 고민을 공유하고 힌트나 조언을 얻으려고 이번에 같이 얘기를 나눠봤는데, 코드의 대한 이해는 나쁘지않지만, 아키텍처 설계가 너무 부실한것 같다는 지적과, 완벽하지 않아도 되니 일단 시작하라는 말이, 그냥 무작정 코드부터 짜면되는게 아니라는 팩트폭력을 받았는데, 이게 많은 인사이트가 되었습니다. 

## 🤔 아쉬움이 남은 일
출퇴근길에 인강을 보고있는데, 지난주는 생각보다 인강보다는 유튜브시청을 많이 한 것 같고, 회사에서 업무에 집중하지않고 개인공부나 카톡에 집중을해서 프로젝트 코드퀄리티가 많이 떨어진 것 같다는 생각을 합니다. 
그리고 이미 끝나가는 프로젝트(SI)다보니 관심을 덜주면서 추가기능에 대한 테스트가 충분히 이뤄지지 않아 고쳐야할부분들이 생긴다는게 크게 반성할 내용입니다 .

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
인강은 이제 기본적인 언어나 API사용법은 어느정도 다 본상태인데, 새로운 인강으로 또 새로운 지식을 공부하기보단 이제 기존에 정리해둔 내용들을 다시한 번 되새기며 내것으로 만들고 기술면접에 대한 준비도 시작해봐야할 것 같습니다.

**Comment by bigsaigon333** [comment]
# 📅 4월 27일~5월 3일의 회고
## 📝 블로그 글 소개
최근 3주간 React를 공부하면서 느꼈던  React  공부방법을 '새로운 기술을 공부하는 방법'으로 추상화해보았습니다. 물론 저에게만 해당되는 내용입니다 ㅎㅎ

## 🤔 아쉬움이 남은 일
생각보다 React에 집중하지 못했던 것 같습니다. 모의면접 준비부터, 뭘 했는지 모를 시간들... 분명 정신없었는데 정신차려보니 일주일이 사라진 듯한 느낌이네요. 매일 일기(데일리 회고)를 작성하자고 다짐했었는데 약 이틀에 한번 정도밖에 작성하지 못하였습니다..

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
- [ ] 운동을 매일매일 하기. 요즘 헬스에 다시 재미를 붙여서 열심히 하고 있습니다. 온라인으로 학습시에는 점심시간을 활용해서, 오프라인으로 학습시에는 일찍 귀가해서 운동 할 생각입니다! 
- [ ] 운동일지도 빠지지 않고 쓰기
- [ ] React 관련 책 빠르게 많이 읽기: 좋은 Input이 많아야 좋은 Output이 조금이라도 나오는 것 같습니다. '깊게 읽기'가 아닌 '빠르게 많이 읽기' 를 목표로 이번주 달려볼 생각입니다. 문제에 맞닥뜨렸을 때, 많은 Input이 일단 주입되었다면 무엇을 꺼내 써야할지를 알 것이므로 그 때에 '깊게 읽기' 또는 깊게 고민하기 단계로 들어갈 예정입니다. 
- [ ] 매일매일 일기(데일리 회고) 작성하기

**Comment by humblEgo** [comment]
# 📅 4월 27일~5월 3일의 회고

## 📝 블로그 글 소개
SLASH 21 중 '결제 시스템의 SDK와 API 디자인' 세션을 듣고 API 디자인 부분만 따로 정리했습니다.

## 💡 인사이트 있었던 일
이력서 피드백을 요청했는데, 제가 생각하는 제 강점이 생각보다 어필되지 않고, 증명도 되지않는 다는 걸 알게 되었습니다.
'증명하기 힘든 것'을 증명하기 위한 방법을 따로 찾아야한다는 걸 느꼈습니다.

## 🤔 아쉬움이 남은 일
알고리즘 문제풀이와 스프링 학습을 가열차게 하고 싶었는데, 시간 배분을 실패했습니다.
자꾸 계획과 다르게 다른 것들(오픈소스 요구사항 분석, SQL, 세미나, 이력서 등)에 시간을 쓴 게 원인이라 생각합니다.
중간에 흥미가는게 생기더라도 하기로 한 것은 딱 해낼 수 있게 제대로 계획하고 지켜야겠습니다.

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
이번 주에 못 했던 알고리즘 문제풀이와 스프링 학습을 가열차게 하고 싶습니다.
- 매일 1시간 확보하여 알고리즘 문제풀이에 시간을 쓰고
- 스프링 학습도 목표한 강의는 다 들어봐야겠습니다.

**Comment by loop-study** [comment]
# 📅 4월 27일~5월 3일의 회고
## 📝 블로그 글 소개
- 4월을 회고하면서 5월 목표를 작성했습니다.
   - TDD, Clean Code With Java 11기 교육 마무리
   - 한동안 고생시킨 이력서도 작성 후, 멘토링 신청
   - 우아한테크코스 Pro 신청하여 프리코스 진행중
   - 공부를 좀 더 효율적이고 체계적으로 진행하기 위해 새로운 방식 도입

## 💡 인사이트 있었던 일
- 회고록을 작성하면, 반성하기도 하고 잘했던 점 스스로를 칭찬하기도 하는데 한달이 아닌 일주일 단위로 진행하자.
- 가이드라인을 따라 이력서를 완성하고 멘토링을 신청했지만, 레거시라 해서 소심할 필요는 없다. 시도한 경험과 도메인을 이해하고 능동적으로 했는지가 더 중요하다.

## 🤔 아쉬움이 남은 일
- 살짝 번아웃이 와서 제대로 공부를 하지 못했다. 2,3개월 뒤에 이직을 시도할 예정이라 시간을 낭비없이 사용했으면 좋겠다.

## 🪜 다음주에 다르게 하고 싶은 것이 있다면?(액션 플랜과 함께)
- 한동안 못했던 운동 다시 시작하기
- 이제 공부(코딩, 독서 등)할 때 스톱워치로 하루에 얼마나 집중하는지 파악예정
- 책이나 인강을 보고나서 리뷰예정 내역을 정리해서 남겨두자.
